package laiofferl;


//unhandled exception
public class ExceptionTest3 {
	static void fun() throws IllegalAccessException, ClassNotFoundException {
		System.out.println("Inside fum()");
		throw new IllegalAccessException("demo");
	}
	public static void main(String args[]) throws IllegalAccessException, ClassNotFoundException {
		fun();
	}
}
